lang/en.js

```diff
@@ -1,14 +1,15 @@
 export default {
   hero: {
-    title: 'Outsite LGBT+ society',
-    subtitle: "Delft's society for queer youth",
+    title: 'Outsite',
```

Comment:
When the title is just 'Outsite', I think it becomes redundant. Maybe we can do something along the lines of:

**Delft's LGBT+ Youth Association**
Where you can be yourself without judgement

(I think this subtitle is a bit cringy, but something like this)

Reply:
Fixed, but with the tagline: "for everyone up to 28 years"